specification/keyvault/resource-manager/readme.md

```diff
@@ -223,6 +223,13 @@ directive:
     - R4015 # The 'MHSMPrivateEndpointConnection' sub-resource doesn't define List as per Network Team's specification.
     - R2005 # The 'ManagedHsms_PurgeDeleted' operation should not return a mix of 202 and syncronous return types (200, 201, 204) as directed by SDK team.
     - R4009 # Vault object is the only one that need to be tracked with SystemData
+- suppress: 
```

Comment:
You have the `api-version` defined inline of get operation but validation uses the common definition at 
https://github.com/Azure/azure-rest-api-specs/blob/ece224eae9f30e64c5e9f28991f45ea4c937bc7a/specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2022-02-01-preview/keyvault.json#L2062

To bypass this is to ignore `where` and one suppression can only include one error code. 

So, it should be like this:
```yml
- suppress: INVALID_REQUEST_PARAMETER
   from: 
   reason: 
- suppress: OBJECT_MISSING_REQUIRED_PROPERTY
   from: xx
   where: 
      - '$.paths./subscriptions/{subscriptionId}/resources.get.parameters[2]'
   reason:
```

Reply:
The suggested suppression directives didn't work. Even the simple suppression directives without any where and from conditions are not working: 
``` yaml
directive:
- suppress:
    - R3016 # The 'release_policy' property for KeyCreateParameters does not support camelCase.
    - R3026 # The 'PrivateEndpointConnection' and 'PrivateLinkResource' sub-resources don't define PATCH as per Network Team's specification.
    - R3025 # The 'PrivateLinkResource' is only accessible via List operation; does not define GET as per Network Team's specification.
    - R4015 # The 'MHSMPrivateEndpointConnection' sub-resource doesn't define List as per Network Team's specification.
    - R2005 # The 'ManagedHsms_PurgeDeleted' operation should not return a mix of 202 and syncronous return types (200, 201, 204) as directed by SDK team.
    - R4009 # Vault object is the only one that need to be tracked with SystemData
    - OBJECT_MISSING_REQUIRED_PROPERTY
    - INVALID_REQUEST_PARAMETER
Could you please take a look and let me know.